idd/Energy+.idd.in

```diff
@@ -372,7 +372,7 @@ Version,
       \format singleLine
   A1 ; \field Version Identifier
       \required-field
-      \default ${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}
+      \default 8.3
```

Comment:
@mjwitte Thanks. I reverted it on my local computer and will make a commit with other potential changes.